cmake/Dependencies.cmake

```diff
@@ -160,9 +160,15 @@ endif()
 
 search_dependency(OPENAL              PACKAGE openal)
 
+find_library(OPENSL_LIBRARY NAMES OpenSLES)
+
+if (OPENSL_LIBRARY)
+  add_dependency(${OPENSL_LIBRARY})
+endif()
+
 if (PLATFORM_EXTENSIONS AND UNIX AND NOT APPLE)
   # Automatic auto-away support. (X11 also using for capslock detection)
-  search_dependency(X11               PACKAGE x11 OPTIONAL)
+  #  search_dependency(X11               PACKAGE x11 OPTIONAL)
```

Comment:
X11 wasn't important, or we get it transitively via something still?

Reply:
I guess we need it. I don't know, everything still works without it, but I've undone it.